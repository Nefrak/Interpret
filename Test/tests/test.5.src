.IPPcode18
DEFVAR GF@counter
DEFVAR GF@help
DEFVAR GF@cislo
DEFVAR GF@char
MOVE GF@cislo int@2
MOVE GF@char string@o
READ GF@counter string 
WRITE GF@counter
STRLEN GF@help GF@counter
WRITE GF@help
GETCHAR GF@help GF@counter int@2
WRITE GF@help
GETCHAR GF@help GF@counter GF@cislo
WRITE GF@help
SETCHAR GF@counter int@2 string@a
WRITE GF@counter
SETCHAR GF@counter GF@cislo GF@char
WRITE GF@counter
TYPE GF@help GF@counter
WRITE GF@help
LABEL end