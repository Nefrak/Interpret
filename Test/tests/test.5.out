textovazprava
13
x
x
teatovazprava
teotovazprava
string

